src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Fixtures/DataSetFileStorageTestFixture.cs

```diff
@@ -300,4 +560,31 @@ public void SetupPatchDataSetVersionCreation()
                 File = null
             });
     }
+
+    private void SetupMockBulkPatchDataSetVersionCreation(DataSetVersion[] dataSetVersions)
+    {
+        for (var i = 0; i < dataSetVersions.Length; i++)
+        {
+            DataSetVersionService.Setup(mock => mock.GetDataSetVersion(
+                    It.IsAny<Guid>(),
```

Comment:
Should we be expecting some specific values here rather than It.IsAny<> here for these various Setups?